<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_close">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 0, 255);</string>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widget_2">
      <attribute name="title">
       <string>SingleMessage</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 255, 15);</string>
       </property>
       <property name="text">
        <string>SPI</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_spn_in">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="text">
        <string>123</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(137, 220, 255);</string>
       </property>
       <property name="text">
        <string>fmi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fmi_in">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>60</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>21</pointsize>
        </font>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_out_can_msg">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>311</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>19</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00 00 00 00 00 00 00 00</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(137, 220, 255);</string>
       </property>
       <property name="text">
        <string>对应的报文</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>711</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(137, 220, 255);</string>
       </property>
       <property name="text">
        <string>原始的报文</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_in_can_msg">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>250</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00 00 00 00 00 00 00 00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>350</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(137, 220, 255);</string>
       </property>
       <property name="text">
        <string>fmi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_fmi_out">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>350</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_spn_out">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>310</y>
         <width>113</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>310</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(191, 255, 15);</string>
       </property>
       <property name="text">
        <string>SPI</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_to_can_info">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 255, 139);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_to_spn_fmi_info">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>410</y>
         <width>531</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(245, 255, 139);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>MultipleMessage</string>
      </attribute>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>251</width>
         <height>191</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 158, 47);</string>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>20</y>
         <width>281</width>
         <height>191</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>240</y>
         <width>701</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>261</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>360</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 158, 47);</string>
       </property>
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_4">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>270</y>
         <width>291</width>
         <height>231</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>130</y>
         <width>101</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 158, 47);</string>
       </property>
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>info</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
